src/platform/release/spec.rs

```diff
@@ -40,16 +40,20 @@ pub struct ReleaseSpec {
 
 impl ReleaseSpec {
     #[instrument(skip_all)]
-    pub fn install(
+    pub fn install<T>(
         &self,
         include_products: &[String],
         exclude_products: &[String],
-    ) -> Result<(), ReleaseInstallError> {
+        namespace: T,
```

Comment:
Namespace could just be a `&str` here, since it's not persisted beyond the function anyway.

Reply:
Changed in df0076f.